### PR TITLE
use createMock() instead of deprecated getMock()

### PR DESCRIPTION
**Pull request by OskarStark**
I am targeting this branch, because this is BC.

### CODE REVIEW

File: Tests/Backend/RuntimeBackendTest.php [hunk 1]
```diff
@@ -15,12 +15,13 @@
 use Sonata\NotificationBundle\Exception\HandlingException;
 use Sonata\NotificationBundle\Model\MessageInterface;
 use Sonata\NotificationBundle\Tests\Entity\Message;
+use Sonata\SeoBundle\Tests\Helpers\PHPUnit_Framework_TestCase;
```
Comment (jordisala1991) [inline Tests/Backend/RuntimeBackendTest.php]: NotificationBundle

Comment (OskarStark) [inline Tests/Backend/RuntimeBackendTest.php]: you are right, thank you..  copy paste fail

File: Tests/Helpers/PHPUnit_Framework_TestCase.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\NotificationBundle\Tests\Helpers;
+
+/**
+ * This is helpers class for supporting old and new PHPUnit versions.
+ *
+ * @todo NEXT_MAJOR: Remove this class when dropping support for < PHPUnit 5.4.
```
Comment (core23) [inline Tests/Helpers/PHPUnit_Framework_TestCase.php]: Do we need this `@todo` here?

**Comment by Koc** [comment]
For what create helper test case? Why not use namespaced version?

**Comment by jordisala1991** [comment]
Because it is not compatible with PHPUnit 4

File: Tests/Helpers/PHPUnit_Framework_TestCase.php (hunk 2 above)
Comment (jordisala1991) [inline Tests/Helpers/PHPUnit_Framework_TestCase.php]: I think it is a copy on all bundles, and should be dropped on master once this is merged. I would not worry about it really

**Comment by jordisala1991** [comment]
@OskarStark did you look at the test? There are some warnings still, not sure if they are from this bundle. Can you take a look at them?

File: Tests/Helpers/PHPUnit_Framework_TestCase.php (hunk 2 above)
Comment (OskarStark) [inline Tests/Helpers/PHPUnit_Framework_TestCase.php]: Yes!

**Comment by OskarStark** [comment]
> @OskarStark did you look at the test? There are some warnings still, not sure if they are from this bundle. Can you take a look at them?

Should be fixed now @jordisala1991

**Comment by greg0ire** [comment]
Thanks @OskarStark !

**Pull request merged**